guidance/models/transformers/_transformers.py

```diff
@@ -146,14 +146,17 @@ def _tokenizer(self, model, **kwargs):
 
     def encode(self, byte_string: bytes) -> Sequence[int]:
         assert isinstance(byte_string, bytes)
-        tokenisation = self._orig_tokenizer(byte_string)
+        # HF tokenisers take in strings apparently
+        tokenisation = self._orig_tokenizer(
+            byte_string.decode(), add_special_tokens=False
+        )
         return tokenisation["input_ids"]
```

Comment:
I think we should standardize on the American spellings for the codebase :) (solely because OpenAI, huggingface, etc. also do so...)